demo/server/src/modules/index.ts

```diff
@@ -1,13 +1,21 @@
 import { EndpointSecurity } from '@raincatcher/auth-passport';
 import { BunyanLogger, Logger } from '@raincatcher/logger';
 import * as express from 'express';
+import EnvironmentConfig, { CloudAppConfig, Config } from '../util/config';
 import { connect as syncConnector } from './datasync/Connector';
 import { router as syncRouter } from './datasync/Router';
+import { init as initKeycloak } from './keycloak';
 import { init as authInit } from './passport-auth';
 
+const appConfig: Config<CloudAppConfig> = new EnvironmentConfig<CloudAppConfig>();
```

Comment:
In seprate PR I made this global variable so you no longer need to create new instances. 
Feel free to merge this. I will refactor it on my branch,